packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -62,78 +62,78 @@
   min-width: var(--LabelMinWidth);
 
   &:not(:only-child) {
-    margin-inline-start: var(--LabelSpaceMarginInlineStart);
+    margin-inline-start: var(--LabelMarginInlineStart);
   }
 }
 
 /* Variants */
 
 .secondaryVariant {
-  border-color: var(--ColorBorderSecondary);
-  background-color: var(--ColorBackgroundSecondary);
-  color: var(--ColorTextSecondary);
+  border-color: var(--SecondaryBorderColor);
+  background-color: var(--SecondaryBackgroundColor);
+  color: var(--SecondaryTextColor);
 
   &:hover {
-    border-color: var(--ColorBorderSecondaryHover);
-    background-color: var(--ColorBackgroundSecondaryHover);
-    color: var(--ColorTextSecondaryHover);
+    border-color: var(--SecondaryHoverBorderColor);
+    background-color: var(--SecondaryHoverBackgroundColor);
+    color: var(--SecondaryHoverTextColor);
   }
 
   &:focus {
-    border-color: var(--ColorBorderSecondaryFocus);
-    background-color: var(--ColorBackgroundSecondaryFocus);
-    color: var(--ColorTextSecondaryFocus);
+    border-color: var(--SecondaryFocusBorderColor);
+    background-color: var(--SecondaryFocusBackgroundColor);
+    color: var(--SecondaryFocusTextColor);
   }
 
   &:disabled {
-    border-color: var(--ColorBorderSecondaryDisabled);
-    background-color: var(--ColorBackgroundSecondaryDisabled);
-    color: var(--ColorTextSecondaryDisabled);
+    border-color: var(--SecondaryDisabledBorderColor);
+    background-color: var(--SecondaryDisabledBackgroundColor);
+    color: var(--SecondaryDisabledTextColor);
   }
 }
 
 .dangerVariant {
-  background-color: var(--ColorBackgroundDanger);
+  background-color: var(--DangerBackgroundColor);
 
   &:hover {
-    border-color: var(--ColorBorderDangerHover);
-    background-color: var(--ColorBackgroundDangerHover);
+    border-color: var(--DangerHoverBorderColor);
+    background-color: var(--DangerHoverBackgroundColor);
   }
 
   &:focus {
-    @include outline(var(--ColorFocusOutlineDanger), var(--FocusOutlineWidth));
+    @include outline(var(--DangerFocusOutlineColor), var(--FocusOutlineWidth));
```

Comment:
I noticed that `FocusOutline` doesnt' align with what I think is the correct implementation of Heirarchy naming. We want the style to be CSS declaration focused as much as possible so this would look like `DangerFocusBoxShadowSpread` and `DangerFocusBoxShadowColor`. Removing the SASS mixin helps make this more clear too!

Reply:
This is an outline-replacement to allow outline styling beyond what the actual property offers for a11y compliance purposes. The mixin sets `outline` to `0` and then creates a pseudo-outline via `box-shadow`. Since `box-shadow`s can be stacked, it's useful to keep this distinct in name and application from other Shadows.

Visually and for Design, there is no "shadow", only an "outline". We use the same approach and mixin across components to keep the output uniform and reduce mistakes. We'll want to remove the Sass implementation at some point, but without using `composes` or similar, I'm not sure how we replicate behavior across Modules.

@andrewberg-okta, did you intend for us to always cleave to the property or should we clarify exceptions where use case trumps the spec?

Reply:
@edburyenegren-okta I understand the details of the implementation yes but I think the decision in front of us is do we create token names the favor what the UI looks like visually (FocusOutline) or how it is implemented (BoxShadow) ?

Reply:
I think it'd behoove us to lean toward use case when the properties don't give enough context. In this case, I wouldn't want a developer to be forced to dig through the style code to determine what the property is trying to achieve.

Given the proposed focus states, the UI deviates even further from the common appearance of `box-shadow` despite it's implementation:

![index](https://user-images.githubusercontent.com/36284167/152023841-8756798c-60f2-4643-937b-09b9283cf2cf.png)

Reply:
As of today in the rest of the cases we're referring to the property directly ( as well as proposed linting for property names ). To move forward I'd like to see that be consistent in all cases for now.

Whether we fill that gap in understanding with an abstraction or documentation feels like a discussion we can have when the new designs for focus have been finalized.

Reply:
but yes in this case in particular we are definitely exposing implementation details that may not matter to a theme author ( or maybe they will which is why i'm torn eg: I have a button on a colored background what is this white border when it's focused?)